packages/orchestrator-ui-components/src/pages/processes/WfoProcessDetail.tsx

```diff
@@ -107,38 +134,40 @@ export const WfoProcessDetail = ({
                     gutterSize="s"
                 >
                     <EuiButton
-                        onClick={(
-                            e: React.MouseEvent<
-                                HTMLButtonElement | HTMLElement,
-                                MouseEvent
-                            >,
-                        ) => {
-                            e.preventDefault();
-                            alert('TODO: Implement retry');
-                        }}
+                        onClick={() =>
+                            showConfirmDialog({
+                                question: t(
+                                    processIsTask
+                                        ? 'retryTaskQuestion'
+                                        : 'retryWorkflowQuestion',
+                                    {
+                                        workflowName:
+                                            processDetail?.workflowName,
+                                    },
+                                ),
+                                confirmAction: () => {
+                                    processDetail?.processId &&
+                                        retryProcess.mutate(
+                                            processDetail.processId,
+                                        );
+                                },
+                            })
+                        }
                         iconType={() => (
                             <WfoRefresh
                                 color={
-                                    buttonsAreDisabled
+                                    retryButtonIsDisabled
                                         ? theme.colors.subduedText
                                         : theme.colors.link
                                 }
                             />
                         )}
-                        isDisabled={buttonsAreDisabled}
+                        isDisabled={retryButtonIsDisabled}
                     >
                         {t('retry')}
                     </EuiButton>
                     <EuiButton
-                        onClick={(
-                            e: React.MouseEvent<
-                                HTMLButtonElement | HTMLElement,
-                                MouseEvent
-                            >,
-                        ) => {
-                            e.preventDefault();
-                            alert('TODO: Implement resume');
-                        }}
+                        onClick={() => alert('TODO: Implement resume')}
```

Comment:
Should implementing this call be part of this PR?

Reply:
Resume is a separate story.